code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -148,7 +148,7 @@ var/global/list/blood_list = list()
 
 
 
-
+//We should really get directional blood streak sprites again --snx
```

Comment:
Those are 'footprints' for wheelchairs. I'm talking about shit like gib1 and gib5, except much worse. I'll get you a picture from another codebase.